app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>TheUltrasNewsroomJan2019</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_pack_tag 'application' %>
+  </head>
+
+  <body>
+    <%= notice || alert %>
+    <%= yield %>
+    <%= link_to "Sign out", destroy_user_session_path, method: :delete %>
```

Comment:
Put a condition on this button to only show when user is logged in, and add a sign in button